integration/testnet/network.go

```diff
@@ -467,29 +485,39 @@ func PrepareFlowNetwork(t *testing.T, networkConf NetworkConfig) *FlowNetwork {
 
 	// create a temporary directory to store all bootstrapping files, these
 	// will be shared between all nodes
-	bootstrapDir, err := ioutil.TempDir(TmpRoot, "flow-integration-bootstrap")
+	bootstrapDir, err := integrationBootstrapDir()
 	require.Nil(t, err)
 
-	fmt.Printf("bootstrapDir: %s \n", bootstrapDir)
+	fmt.Printf("BootstrapDir: %s \n", bootstrapDir)
 
 	root, result, seal, confs, err := BootstrapNetwork(networkConf, bootstrapDir)
 	require.Nil(t, err)
 
 	flowNetwork := &FlowNetwork{
-		t:                  t,
-		cli:                dockerClient,
-		config:             networkConf,
-		suite:              suite,
-		network:            network,
-		Containers:         make(map[string]*Container, nNodes),
-		ConsensusFollowers: make(map[flow.Identifier]consensus_follower.ConsensusFollower, len(networkConf.ConsensusFollowers)),
-		AccessPorts:        make(map[string]string),
-		root:               root,
-		seal:               seal,
-		result:             result,
-		bootstrapDir:       bootstrapDir,
-		allPorts:           make(map[string]struct{}),
+		t:                          t,
+		cli:                        dockerClient,
+		config:                     networkConf,
+		suite:                      suite,
+		network:                    network,
+		Containers:                 make(map[string]*Container, nNodes),
+		ConsensusFollowers:         make(map[flow.Identifier]consensus_follower.ConsensusFollower, len(networkConf.ConsensusFollowers)),
+		AccessPorts:                make(map[string]string),
+		AccessPortsByContainerName: make(map[string]string),
+		root:                       root,
+		seal:                       seal,
+		result:                     result,
+		BootstrapDir:               bootstrapDir,
+	}
+
+	// check that at-least 2 full access nodes must be configure in your test suite
+	// in order to provide a secure GRPC connection for LN & SN nodes
+	accessNodeIDS := make([]string, 0)
+	for _, n := range confs {
+		if n.Role == flow.RoleAccess && !n.Ghost {
+			accessNodeIDS = append(accessNodeIDS, n.NodeID.String())
+		}
```

Comment:
It seems like if we use the `*` option, to use all available AN IDs, we can avoid this check (for non-Mainnet networks).
https://github.com/onflow/flow-go/blob/e3627a7ada8d2a8178e7221fa05640bbd9270983/cmd/util/cmd/common/validation.go#L17-L25

I think for the integration tests, we should make use of this. We'll have one less container eating resources and a bit less complexity in the test setup code.

Reply:
Yes great idea !

Reply:
https://github.com/onflow/flow-go/pull/1603/commits/e15d3c983f39bc37be9c3610359504c130ad56a1